.ci/scripts/release/prepare.sh

```diff
@@ -2,11 +2,16 @@
 
 # update apt repositories and install missing utilities
 # add GitHub packages Debian repository for gh
-curl -fsSL https://cli.github.com/packages/githubcli-archive-keyring.gpg | dd of=/usr/share/keyrings/githubcli-archive-keyring.gpg
-echo "deb [arch=$(dpkg --print-architecture) signed-by=/usr/share/keyrings/githubcli-archive-keyring.gpg] https://cli.github.com/packages stable main" | \
-      tee /etc/apt/sources.list.d/github-cli.list > /dev/null
+
 apt update
-apt install -y gpg gh
+apt install -y gpg git
+
+# install using apt fails due to https://github.com/cli/cli/issues/6175 Workaround is to get it from git release
+curl -L https://github.com/cli/cli/releases/download/v2.14.7/gh_2.14.7_linux_amd64.deb --output /tmp/gh.deb
+dpkg -i /tmp/gh.deb
+
+# Have to set the safe directory otherwise git commands fails with fatal: detected dubious ownership in repository at '/home/jenkins/agent/workspace/zeebe-release'
```

Comment:
I'm not sure where this is necessary, or when did this break :thinking: We've been using git for a while. Anyway :+1:

Reply:
Script fails without this. Now sure why it fails now, and was working before.